Source/Lib/Codec/EbMotionEstimation.c

```diff
@@ -799,7 +799,7 @@ static void PU_HalfPelRefinement(
         *pBestSsd = (EB_U32) SpatialFullDistortionKernel_funcPtrArray[!!(ASM_TYPES & PREAVX2_MASK)][Log2f(puWidth) - 2](
             &(contextPtr->lcuSrcPtr[puLcuBufferIndex]),
             contextPtr->lcuSrcStride,
-            &(refBuffer[ySearchIndex * refStride + xSearchIndex]),
+            refBuffer + (ySearchIndex * (EB_S32)refStride + xSearchIndex),
```

Comment:
@Austin-Hu  - did you find this change made any difference?

Reply:
It just works around invalid addressing for some LCUs, and actually it's unecessary with the 2nd commit applied. So it will be removed once the solution is finalized.